### DISCUSSION TITLE
Who is your favorite actor?

### DISCUSSION THREAD

**Discussion by sy-zvjkqv** [NAIST-SE/AttractingContributors#882] [General]
あなたの好きな俳優を教えてください

my repo: https://github.com/sy-zvjkqv/AttractingContributors

**Comment by hiromuotsubo** [comment]
リストにJim Carreyを追加するプルリクエストを送りました。
私のところにもプルリクエストしてもらえないでしょうか？あなたの好きな国 or 行ってみたい国を追加してほしいです。
https://github.com/Tsubonee/AttractingContributors-country

I made a pull request to add "Jim Carrey" to your list.
Please add your favorite country or country you want to visit.
https://github.com/Tsubonee/AttractingContributors-country
